src/components/articlepreview.js

```diff
@@ -29,6 +32,7 @@ const ArticlePreview = (props) => {
           </Link>
         </h3>
         <Byline author={author} date={date} identifier={identifier}/>
+        <Tags tags={tags}/>
```

Comment:
I'd rather move them below the article or even drop them from the front page. They are a bit distracting and maybe not that important?

Reply:
removed from the front page